dagger/config_finder/config_processor.py

```diff
@@ -21,18 +21,12 @@ def __init__(self, config_finder: ConfigFinder):
         self._config_finder = config_finder
         self._task_factory = TaskFactory()
 
-    @staticmethod
-    def _load_yaml(yaml_path):
-        with open(yaml_path, "r") as stream:
-            try:
-                config = yaml.safe_load(stream)
-            except yaml.YAMLError as exc:
-                _logger.exception("Couldn't read config file {}", yaml_path, exc)
-                exit(1)
-        return config
+    def _load_yaml(self, yaml_path):
+        config_dict = EnvYAML(yaml_path).export()
+        config_dict = self.localize_params(config_dict)
+        return config_dict
 
-    @staticmethod
```

Comment:
I don't see where you are using self in this function so not sure why it became non static